python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -590,6 +590,11 @@ def _get_actions(self, data):
                 logger.debug("Querying: %s", lookup_hash)
 
                 pc_data["cached_data"] = []
+
+                # If the config doesn't support the current entity_type we don't need to cache it
+                if not lookup_hash:
```

Comment:
I'm being picky, and this is personal preference so honestly feel free to take it or leave it, but I prefer `if lookup_has is None:` as it is more explicit about what exactly we are checking.

Also about line 589 `lookup_hash = pc_data["lookup_hash"]`.

I know this is the first thing that is defined in the phase 1 loop, but it didn't mean I had to go and check to see if it was always defined. Maybe there is a good reason not to, but I usually go with `pc_data.get("lookup_hash")` and then I wouldn't need to ensure that it is defined.